.changeset/cyan-fireants-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'highlight.run': patch
+---
+
+fix `H.getSession*` methods using stale session ID
```

Comment:
@Vadman97 should I do a release as part of this PR since we want the change out immediately?

Reply:
sure, especially since we don't have any other changesets in `.changesets` currently, so the release would only have this pr. you can run `yarn changeset version` on this branch (also `yarn reflame-build`) and push that - once merged to main that should release the npm package

Reply:
Done!